### ISSUE TITLE
Object in embedded <ng-content> is not attached to scene

### ISSUE THREAD

**Issue by makimenko** [makimenko/angular-template-for-threejs#87]
**Describe the bug**
AbstractObject3D in the ng-content is not visible for parent. As result it's not added to the scene.

For example:
ComponentA:
```
<atft-scene>
     ....
     <ng-content></ng-content>
</atft-scene>
```

ComponentB:
```
<component-a>
   <atft-axes-helper size=200></atft-axes-helper>
   ...
</component-a>
```

=> atft-axes-helper is not added to the scene.

**To Reproduce**
https://makimenko.github.io/angular-template-for-threejs/storybook/?path=/story/bugs--87
(see Notes in Storybook)

**Actual**
atft-cylinder-mesh not added to the scene

**Expected**
atft-cylinder-mesh should be added to the scene

**Comment by ComFreek** [comment]
Perhaps related to https://github.com/angular/angular/issues/16299 and linked StackOverflow question from there: http://stackoverflow.com/questions/43600969/angular2-contentchildren-not-populated-inside-parent-ng-content/43601229#43601229

Perhaps some dirty [workaround via `ngAfterContentInit`](https://stackoverflow.com/a/49287423/603003) could work as well.

**Comment by makimenko** [comment]
partly related to #168

**Comment by makimenko** [comment]
![image](https://user-images.githubusercontent.com/11466819/95472558-df83e800-098b-11eb-8a3a-efdd78a595a4.png)
Implemented in #307
